doc/glossary.rst

```diff
@@ -14,10 +14,10 @@
                 was designed specifically for use with Ceph. BlueStore was
                 introduced in the Ceph Kraken release. In the Ceph Luminous
                 release, BlueStore became Ceph's default storage back end,
-                supplanting FileStore. Unlike :term:`filestore`, BlueStore
-                stores objects directly on Ceph block devices without any file
-                system interface. Since Luminous (12.2), BlueStore has been
-                Ceph's default and recommended storage back end.
+                supplanting FileStore. BlueStore stores objects directly on 
+                Ceph block devices without any file system interface. 
```

Comment:
Not a new thing but _Ceph block devices_ is  a bit confusing.

Reply:
I'm flagging this as "unfinished business" and will deal with "Ceph block devices" later. (This means that I am making a note of it and will return to it later in the week or month, but I intend now to merge this PR to main and to backport the commit associated with it to Reef (and only to Reef, because of Quincy and Pacific's support for FileStore)).

Reply:
In a future PR, I will change "Ceph block device" to "block device".